t/08-autotest.t

```diff
@@ -451,7 +451,7 @@ subtest loadtestdir => sub {
     stderr_like {
         autotest::loadtestdir('tests');
     } qr/debug.*scheduling/, 'loadtestdir is scheduling successfully';
-    ok exists $autotest::tests{'tests-boot'}, 'boot.pm loaded';
```

Comment:
Are we losing test coverage for Python here?

Reply:
Why for python?